superagi/helper/github_helper.py

```diff
@@ -9,7 +9,8 @@
 from superagi.types.storage_types import StorageType
```

Comment:
Can we now on put a helper in the tool folder making a new folder like inside GitHub a folder named helper and inside that all helper logic as we might need to extract out all tools in the marketplace tool repo?